src/pages/changelog/index.md

```diff
@@ -11,6 +11,9 @@ contributors:
   - https://github.com/padmkris123
 ---
 # Changelog
+## InDesign v19.2
+#### Updated
+- **[BREAKING CHANGE]** UXP logs and console.log paths have changed.
```

Comment:
Please add the link for debug tutorial here so that the developers can read more about the details.

Reply:
Done.